src/controllers/restful/PaymentController.ts

```diff
@@ -73,21 +73,24 @@ AsControllerFunction<GetPaymentInfoT> = (
   pagoPAConfig,
   pagoPAClient,
   pagoPAClientNm3
-  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type,max-lines-per-function
 ) => async params => {
+  const clientId = params["x-Client-Id"];
+
   // Validate rptId (payment identifier) provided by BackendApp
   const errorOrRptId = RptIdFromString.decode(params.rpt_id_from_string);
```

Comment:
to add a check to verify if `clientId` is set with `CLIENT_CHECKOUT` or `CLIENT_IO`.
If it is not set correctly we can return 400, what do you think?